### PR TITLE
Port Scan - Internal and External Source - Block IP V3

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-2690

## Description
Replaced "Block IP - V2" with "Block IP - V3" in the "Port Scan - Internal Source" playbook.

## Screenshots
Port Scan - Internal Source playbook:
Port_Scan_-_Internal_Source

Port Scan - External Source playbook:
Port_Scan_-_External_Source

### CODE REVIEW

File: Packs/PortScan/Playbooks/Port_Scan_-_Internal_Source_README.md [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
+| InternalSourceIPs | The internal IP address\(es\) that performed the port scan. In order to properly respond to internal scans, either this or the SourceHostnames input has to be filled with data. |  | Optional |
+| SourceHostnames | The source hostname\(s\) that performed the port scan. In order to properly respond to internal scans, either this or the InternalSourceIPs input has to be filled with data. |  | Optional |
+| SourceUsernames | The Active Directory username\(s\) that were used in the port scan attack. |  | Optional |
+| InvolvedFileMD5 | MD5 hashes of files that were involved in the port scan incident. |  | Optional |
+| InvolvedFileSHA1 | SHA1 hashes of files that were involved in the port scan incident. |  | Optional |
+| InvolvedFileSHA256 | SHA256 hashes of files that were involved in the port scan incident. |  | Optional |
+| DBotScore | All the DBotScores that were calculated either automatically by auto-reputation, or using specific tasks, when the incident was ingested. This is used to calculate the incident severity at a later stage. | DBotScore.None | Optional |
```
Comment (idovandijk) [inline Packs/PortScan/Playbooks/Port_Scan_-_Internal_Source_README.md:43]: Notice the `.None` in this README file. This is not your fault, but it looks like a bug in the SDK `generate-docs` command. Please open a Jira ticket about this. Attach to the ticket the relevant playbook YML file and the readme MD file so the dev team can debug this.

File: Packs/PortScan/Playbooks/Port_Scan_-_External_Source_README.md [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
-
-## Playbook Inputs
----
-
-| **Name** | **Description** | **Default Value** | **Source** | **Required** |
-| --- | --- | --- | --- | --- |
-| BlockAttackerIP | Whether attacking IPs should be automatically blocked using firewalls. | False |  | Required |
-| SourceExternalIPs | The external IP address\(es\) that initiated the port scan. |  |  | Optional |
-| BlockMaliciousDomains | In the event that reverse IP lookup is performed, and a malicious domain is found, setting this to True will automatically block the malicious domains. If set to False, an analyst can manually block the domains. | False |  | Required |
-| DBotScore | All the DBotScores that were calculated, either automatically by auto\-reputation or using specific tasks, when the incident was ingested. This is used to calculate the incident severity at a later stage. | None | DBotScore | Optional |
```
Comment (idovandijk) [inline Packs/PortScan/Playbooks/Port_Scan_-_External_Source_README.md:32]: See other comment about the "None" here

**Review by idovandijk — APPROVED** [review]
Excellent job. Please see my comment.

File: Packs/PortScan/Playbooks/Port_Scan_-_External_Source_README.md (hunk 2 above)
Comment (TalNos) [reply Packs/PortScan/Playbooks/Port_Scan_-_External_Source_README.md:32]: ### I open a Jira ticket. Should I change it manually in the meantime?

Comment (idovandijk) [reply Packs/PortScan/Playbooks/Port_Scan_-_External_Source_README.md:32]: Yes :)

**Pull request merged**